packages/back-end/types/datasource.d.ts

```diff
@@ -12,6 +12,7 @@ import { MetricType } from "./metric";
 import { MssqlConnectionParams } from "./integrations/mssql";
 
 export type DataSourceType =
+  | "growthbook_clickhouse"
```

Comment:
Aren't we here just overlooking the fact that this should be database type clickhouse, schema growthbook? Not blocking for now.